{{cookiecutter.project_shortname}}/templates/invenio_app_ils/notifications/{{cookiecutter.project_shortname}}_footer.html

```diff
@@ -0,0 +1,25 @@
+{% raw %}
+{% block footer_plain %}
+Check your library loans and requests in your profile page <{{ spa_routes.HOST }}{{ spa_routes.PATHS["profile"] }}>.
+
+Kind regards,
+
+CERN Library
+
+Location: building 52/1-052 on the Meyrin campus
+Opening hours (https://catalogue.library.cern/opening-hours)
+Email: library.desk@cern.ch
+{% endblock %}
+
+{% block footer_html %}
+<br/><br/>
+Check your library loans and requests in your <a href="{{ spa_routes.HOST }}{{ spa_routes.PATHS["profile"] }}">profile page</a>.
+<br/><br/>
+Kind regards, <br/><br/>
+
+CERN Library <br/>
+Location: building 52/1-052 on the Meyrin campus <br/>
+<a href="https://catalogue.library.cern/opening-hours"> Opening hours</a> <br/>
+Email: library.desk@cern.ch <br/>
```

Comment:
Please replace all `library.desk@cern.ch` with `library.desk@{{ cookiecutter.project_shortname }}` (to be tested)